infra/events/src/main/java/com/simprints/infra/events/event/local/migrations/EventMigration15to16.kt

```diff
@@ -0,0 +1,61 @@
+package com.simprints.infra.events.event.local.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import com.simprints.core.tools.extentions.getStringWithColumnName
+import com.simprints.infra.logging.Simber
+
+internal class EventMigration15to16 : Migration(15, 16) {
+
+    override fun migrate(db: SupportSQLiteDatabase) {
+        Simber.d("Migrating room db from schema 15 to schema 16.")
+        renameFaceLicenseErrors(db)
+        Simber.d("Migration from schema 15 to schema 16 done.")
+    }
+
+    // rename FACE_LICENSE_MISSING and FACE_LICENSE_INVALID to LICENSE_MISSING and LICENSE_INVALID
+    private fun renameFaceLicenseErrors(database: SupportSQLiteDatabase) {
+        // select all alert AlertScreenEvent with type FACE_LICENSE_MISSING or FACE_LICENSE_INVALID
```

Comment:
Is it enough that we rename the type of the AlertScreenEvent? Did you test it out?